lib/shared.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
```

Comment:
Please no shared.js. Create a class instead. You could create a RandomGenerator class.